tests/bud.bats

```diff
@@ -1981,6 +1981,32 @@ _EOF
   test -s $root/etc/passwd
 }
 
+@test "build using --layer-label and test labels on intermediate images" {
+  # Remove all images so no intermediate images are present
+  run_buildah rmi --all -f
+  _prefetch alpine
+  run_buildah --version
+  local -a output_fields=($output)
+  buildah_version=${output_fields[2]}
+  label="l_$(random_string)"
+  labelvalue="v_$(random_string)"
+
+  run_buildah build --no-cache --layers --layer-label $label=$labelvalue --layer-label emptylabel $WITH_POLICY_JSON -t exp -f $BUDFILES/simple-multi-step/Containerfile
+
+  # Final image must not contain this label
+  run_buildah inspect --format '{{ index .Docker.Config.Labels "'$label'"}}' exp
+  expect_output "" "label on actual image"
+
+  # All intermediate images must have this label
```

Comment:
With your latest change, this comment is now wrong, it should be `these labels`, plural.